chat-plugins/gold.js

```diff
@@ -1327,3 +1306,21 @@ Gold.pluralFormat = function(length, ending) {
 	if (isNaN(Number(length))) return false;
 	return (length == 1 ? '' : ending);
 }
+Gold.regdate = function(name) {
+	name = toId(name);
+	if (regdateCache[name]) return regdateCache[name];
+	request('http://pokemonshowdown.com/users/' + name + '.json', function (error, response, body) {
+		var data = JSON.parse(body);
+		var date = data['registertime'];
+		if (date !== 0 && date.toString().length < 13) {
+			while (date.toString().length < 13) {
+				date = Number(date.toString() + '0');
+			}
+		}
+		if (date !== 0) {
+			regdateCache[name] = date;
+			saveRegdateCache();
+		}
+	});
+	return regdateCache[name] || 0;
```

Comment:
this could probably return "is not registered" if it's 0

on a sidenote, refer to the pastebin I posted here